CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ cmake_minimum_required ( VERSION 3.13 )
 # 3.11 because COMPATIBILITY SameMinorVersion in CMakePackageConfigHelpers
 # 3.13.5 because it is the latest supported in Windows XP
 
+set ( FLUIDSYNTH_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR} )
+set ( FLUIDSYNTH_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR} )
+
```

Comment:
In my opinion, there is no need to create those variables, when there are already two created:
* `FluidSynth_SOURCE_DIR`
* `FluidSynth_BINARY_DIR`
These variables are created for every cmake project, when the command [project()](https://cmake.org/cmake/help/v3.13/command/project.html) is called.